web/client/components/tutorial/Tutorial.jsx

```diff
@@ -7,12 +7,12 @@
  */
 const PropTypes = require('prop-types');
 const React = require('react');
-const Joyride = require('react-joyride').default;
+const Joyride = require('@geosolutions/react-joyride').default;
```

Comment:
Also for this one we can add an alias (in general I would add one for every @geosolutions package we forked, except for those used only by the tools, for example mocha is not needed, I think)

Reply:
Well, I think we can remove the changes to postInstall if we add mocha as an alias. 
Am i correct?

Reply:
i just verified. We need to keep changes to postinstall for mocha

Reply:
yes, I think karma is a different beast